src/components/App/test.tsx

```diff
@@ -1,17 +1,19 @@
 import React from "react"
 import ReactDOM from "react-dom"
 import { App } from "."
-import { shallowRender } from "../../utils/shallowRender"
+import { cleanup, render } from "@testing-library/react"
 
 describe("App", () => {
+  beforeEach(cleanup)
+
   it("renders", () => {
     const div = document.createElement("div")
     ReactDOM.render(<App />, div)
     ReactDOM.unmountComponentAtNode(div)
   })
 
   it("renders expected structure", () => {
-    const received = shallowRender(<App />)
-    expect(received).toMatchSnapshot()
+    const { container } = render(<App />)
+    expect(container).toMatchSnapshot()
```

Comment:
@roblafeve Does it make sense now that we have removed default snapshot tests to still include one as an example, or should we change this over to a test similar to the generated tests?

Reply:
@blaketarter good question—I'm not sure 😬